### PR TITLE
Fix typo.

### PR DESCRIPTION
**Pull request by BillyZac**

### CODE REVIEW

**Comment by bterlson** [comment]
Thanks!

File: spec.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
...
   <p>ECMAScript was originally designed to be a <em>Web scripting language</em>, providing a mechanism to enliven Web pages in browsers and to perform server computation as part of a Web-based client-server architecture. ECMAScript is now used to provide core scripting capabilities for a variety of host environments. Therefore the core language is specified in this document apart from any particular host environment.</p>
-  <p>ECMAScript usage has moved beyond simple scripting and it is now used for the full spectrum of programming tasks in many different environments and scales. As the usage of ECMAScript has expanded, so has the features and facilities it provides. ECMAScript is now a fully featured general propose programming language.</p>
+  <p>ECMAScript usage has moved beyond simple scripting and it is now used for the full spectrum of programming tasks in many different environments and scales. As the usage of ECMAScript has expanded, so has the features and facilities it provides. ECMAScript is now a fully featured general purpose programming language.</p>
```
Comment (mathiasbynens) [inline spec.html]: While we’re at it, shouldn’t this be “general-purpose” (with a hyphen)?

Comment (Alhadis) [inline spec.html]: I'd also change `fully featured` to `full-featured` too.

Comment (ljharb) [inline spec.html]: `fully featured` sounds more correct to me.

Comment (Alhadis) [inline spec.html]: LOL @ the thumbs down reaction. Here, have one back.

**Pull request merged**